tests/org.jboss.tools.cdi.bot.test/src/org/jboss/tools/cdi/bot/test/wizard/template/ProjectWithCDITemplate.java

```diff
@@ -209,26 +205,47 @@ public void createUtilityProject() {
 		uw.open();
 		UtilityProjectFirstPage up = new UtilityProjectFirstPage(uw);
 		up.setProjectName(PROJECT_NAME);
-		up.activateFacet("1.8", "Java");
+		if (isJavaLE8() || CDIVersion.equals("1.0")) {
+			up.activateFacet("1.8", "Java");
+		} else {
+			up.activateFacet("11", "Java");
+		}
 		uw.finish();
+		new WaitUntil(new JobIsRunning(), TimePeriod.DEFAULT, false);
+		new WaitWhile(new JobIsRunning(), TimePeriod.LONG);
+		setWorkspaceDefaultJREForJava8();
```

Comment:
Same question as above. And below as well.

Reply:
Fixed.